src/index.js

```diff
@@ -8,129 +8,141 @@ import parseOpenApi3Documentation from "@api-platform/api-doc-parser/lib/openapi
 import { version } from "../package.json";
 import generators from "./generators";
 
-program
-  .version(version)
-  .description(
-    "Generate apps built with Next, Nuxt, Quasar, React, React Native, Vue or Vuetify for any API documented using Hydra or OpenAPI"
-  )
-  .usage("entrypoint outputDirectory")
-  .option(
-    "-r, --resource [resourceName]",
-    "Generate CRUD for the given resource"
-  )
-  .option(
-    "-p, --hydra-prefix [hydraPrefix]",
-    "The hydra prefix used by the API",
-    "hydra:"
-  )
-  .option("--username [username]", "Username for basic auth (Hydra only)")
-  .option("--password [password]", "Password for basic auth (Hydra only)")
-  .option("--bearer [bearer]", "Token for bearer auth (Hydra only)")
-  .option(
-    "-g, --generator [generator]",
-    'The generator to use, one of "next", "nuxt", "quasar", "react", "react-native", "typescript", "vue", "vuetify"',
-    "next"
-  )
-  .option(
-    "-t, --template-directory [templateDirectory]",
-    "The templates directory base to use. Final directory will be ${templateDirectory}/${generator}",
-    `${__dirname}/../templates/`
-  )
-  .option(
-    "-f, --format [hydra|openapi3|openapi2]",
-    '"hydra", "openapi3" or "openapi2"',
-    "hydra"
-  )
-  .option(
-    "-s, --server-path [serverPath]",
-    "Path to express server file to allow route dynamic addition (Next.js generator only)"
-  )
-  .parse(process.argv);
+async function main() {
+  program
+    .version(version)
+    .description(
+      "Generate apps built with Next, Nuxt, Quasar, React, React Native, Vue or Vuetify for any API documented using Hydra or OpenAPI"
+    )
+    .usage("entrypoint outputDirectory")
+    .option(
+      "-r, --resource [resourceName]",
+      "Generate CRUD for the given resource"
+    )
+    .option(
+      "-p, --hydra-prefix [hydraPrefix]",
+      "The hydra prefix used by the API",
+      "hydra:"
+    )
+    .option("--username [username]", "Username for basic auth (Hydra only)")
+    .option("--password [password]", "Password for basic auth (Hydra only)")
+    .option("--bearer [bearer]", "Token for bearer auth (Hydra only)")
+    .option(
+      "-g, --generator [generator]",
+      'The generator to use, one of "next", "nuxt", "quasar", "react", "react-native", "typescript", "vue", "vuetify"',
+      "next"
+    )
+    .option(
+      "--custom-generator [customGenerator]",
```

Comment:
Can't we merge this option with the existing `generator` option?

(If the module exists, load it)